### PR TITLE
Updating CLI shared framework dependencies to 1.0.3-servicing-004584

### PR DESCRIPTION
**Pull request by livarcocc**
@piotrpMSFT @krwq @jgoshi @leecow @gkhanna79

### CODE REVIEW

**Comment by gkhanna79** [comment]
@mmitche We see the following failure here (and in Core-Setup):

```
13:26:19 Building Docker Container using Dockerfile: scripts/docker/ubuntu.14.04
13:26:19 Cannot connect to the Docker daemon. Is the docker daemon running on this host?
13:26:19 Build step 'Execute shell' marked build as failure
```

**Comment by livarcocc** [comment]
@dotnet-bot Test CentOS7.1 x64 Debug Build
@dotnet-bot Test Ubuntu x64 Release Build

File: TestAssets/ProjectModelServer/DthTestProjects/src/BrokenProjectPathSample/project.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.App": "1.0.2",
+    "Microsoft.NETCore.App": "1.0.3-servicing-004584-00",
```
Comment (piotrpMSFT) [inline TestAssets/ProjectModelServer/DthTestProjects/src/BrokenProjectPathSample/project.json]: Imagine this change once we move to csproj... one line of code... :)

**Comment by piotrpMSFT** [comment]
:shipit:

**Pull request merged**